### ISSUE TITLE
Review existing documents

### ISSUE THREAD

**Issue by mirdaki** [ossn/project-template#1]
The documents could be simplified, structured better, or just written better, so it'd be great for people to [make issues](https://github.com/ossn/project-template/issues) about changes to documents.

**Comment by nik72619c** [comment]
Could I work on this one?

**Comment by mirdaki** [comment]
@nik72619c Yes! I've actually made a [new version of this repo](https://github.com/ossn/club-project-template) (this one was just a copy of the UF OSC version, so it wasn't getting any updates). I'll add this issue on that one now, if you want to start working on it there.
